<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="L-GDZ6152" name="com.digi.selenium.app.Stores_Orders.Store_Orders_Postpaid" tests="1" failures="1" timestamp="10 Nov 2015 04:45:47 GMT" time="0.000" errors="0">
  <testcase name="validateStoreNordersTotalAmount" time="0.000" classname="com.digi.selenium.app.Stores_Orders.Store_Orders_Postpaid">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.digi.selenium.util.common.PageNavigation.handleLink_ByText(PageNavigation.java:890)
	at com.digi.selenium.util.common.PageNavigation.signInWithMSSIDN_stage(PageNavigation.java:950)
	at com.digi.selenium.util.Stores_Orders.Stores_Orders_Postpaid_Util.setUp(Stores_Orders_Postpaid_Util.java:43)
	at com.digi.selenium.app.Stores_Orders.Store_Orders_Postpaid.initDriver(Store_Orders_Postpaid.java:25)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:564)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:213)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:138)
	at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:175)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:107)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:207)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:178)
]]>
    </failure>
  </testcase> <!-- validateStoreNordersTotalAmount -->
</testsuite> <!-- com.digi.selenium.app.Stores_Orders.Store_Orders_Postpaid -->
